R/mwrite.R

```diff
@@ -120,6 +120,8 @@ mwrite_model_to_list <- function(x) {
   code <- modelparse(code, comment_re = character(0))
   code <- lapply(code, trimws, which = "right")
   
+  names(code) <- toupper(names(code))
```

Comment:
Is it worth adding something like below as a test?

```R
test_that("mwrite normalizes block names to upper case", {
  skip_if_not_installed("yaml")

  ucode <- '$CMT DEPOT CENT
$PKMODEL ncmt=1, depot=TRUE
$MAIN
double CL = 1
double V = 20
double KA = 1'
  umod <- mcode("test-case", ucode, compile = FALSE)
  utemp <- tempfile()
  mwrite_yaml(umod, file = utemp)

  lcode <- '$cmt DEPOT CENT
$pkmodel ncmt=1, depot=TRUE
$main
double CL = 1
double V = 20
double KA = 1'
  lmod <- mcode("test-case", lcode, compile = FALSE)
  ltemp <- tempfile()
  mwrite_yaml(lmod, file = ltemp)

  expect_identical(readLines(utemp), readLines(ltemp))
})
```

Reply:
Thanks @kyleam; I had a simple test in place, but looks like I didn't push that. Will implement the proposed test and make sure everything is pushed.